test/README.md

```diff
@@ -0,0 +1,117 @@
+# Tests
+
+###abort
+
+Tests for when the `--abort-on-uncaught-exception` flag is used.
+
+| Runs on CI |
+|:----------:|
+| No         |
+
+###addons
```

Comment:
Please add a space after the hashes. Various markdown parsers fail to render such headings otherwise.